tests/airdrop_lamports.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+for i in `seq 1 6`
+do
+    solana airdrop 500.0
+    if [ $? -eq 0 ]
+    then
+        break
+    else
+        if [ $i -eq 5 ]
+        then
+            echo "Airdrop failed after trying 5 times, giving up."
+            exit 1
+        else
+            echo "Airdrop failed, awaiting 2s before asking again..."
+            sleep 2
+        fi
+    fi
+done
```

Comment:
This borders on what I think is still acceptable in Bash before I would reach for a sane language :see_no_evil: But in this case it’s fine.